### PR TITLE
Bump ecmaVersion 6 to 8

### PR DESCRIPTION
**Pull request by stonebuzz**
As seen here
https://github.com/glpi-project/glpi/pull/11487

Bump ecmaversion to 7 for javascript ```async function```

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

**Comment by cconard96** [comment]
ECMA version gets bumped to 2021 (ES12) in 10.1. Something I noticed trying to change the parser options to ES 7 for async function support is that ESLint still marked it as invalid. I has to remove the es6 environment option and replace it with es2021 for it to accept it. I didn't see any other `es` environment option.

This could have just been an issue on my end though.

**Comment by stonebuzz** [comment]
@orthagh, @trasher , @cedric-anne 

Should I change to the ES12 ?

File: .eslintrc.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "_nx": true
     },
     "parserOptions": {
-        "ecmaVersion": 6
+        "ecmaVersion": 7
```
Comment (cedric-anne) [inline .eslintrc.json:30]:
```suggestion
        "ecmaVersion": 8
```

**Comment by cedric-anne** [comment]
> @orthagh, @trasher , @cedric-anne
> 
> Should I change to the ES12 ?

I changed to ES8.

ES8 was supported on Edge since v15, that was released in 2017. Next versions are only available since switch to Chromium engine, which has been done in 2020.

I think it is more secure to ensure compatibility to ES8 in GLPI 10.0.

**Pull request merged**